Terminal.Gui/Windows/FileDialog2.cs

```diff
@@ -0,0 +1,2118 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using NStack;
+using Terminal.Gui.Trees;
+using static System.Environment;
+using static Terminal.Gui.OpenDialog;
+
+namespace Terminal.Gui {
+
+	/// <summary>
+	/// Modal dialog for selecting files/directories.  Has auto-complete and expandable
```

Comment:
Please, please, please stop putting two spaces after periods. I know it's a hard habit to break. 💕

Reply:
Ill do a find and replace ;) its subconcious

Reply:
Fixed